src/components/EventCard/index.js

```diff
@@ -14,6 +15,22 @@ import styles from './styles';
 import { getLocalDateTimeString, getEndTimeCountDownString } from '../../helpers/utility';
 
 class EventCard extends React.PureComponent {
+  static propTypes = {
```

Comment:
why static?

Reply:
propTypes are static items as they are all declared the same way across all instances. the previous method of declaring them is the same way:
EventCard.propTypes is declaring them in a static way.